sdk/spring/azure-spring-cloud-stream-binder-test/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Release History
 
 ## 2.12.0-beta.1 (Unreleased)
+This release is compatible with Spring Boot 2.5.0-2.5.4, 2.6.0-2.6.1, and Spring Cloud 2021.0.0.
```

Comment:
Good. This version does not support Spring Cloud 2020.0.

Reply:
How about the modules that depend on this one?
![image](https://user-images.githubusercontent.com/13167207/147047773-6623f3c5-4d60-4acf-aafa-00212083c7a8.png)

Reply:
Tested the three binders，using spring boot 2.5.7 spring cloud 2020.0.5 and spring boot 2.6.1 spring cloud 2021.0.0, they all work normally.